web-api/tests/Kerbero.Integration.Tests/SmartLockKeys/CreateSmartLockKeyIntegrationTests.cs

```diff
@@ -0,0 +1,64 @@
+using System.Net;
+using System.Net.Http.Json;
+using FluentAssertions;
+using Flurl.Http.Testing;
+using Kerbero.Domain.SmartLockKeys.Dtos;
+using Kerbero.WebApi;
+using Kerbero.WebApi.Models.Requests;
+
+namespace Kerbero.Integration.Tests.SmartLockKeys;
+
+public class CreateSmartLockKeyIntegrationTests: IDisposable
+{
+	private readonly KerberoWebApplicationFactory<Program> _application;
+	private readonly HttpTest _httpTest;
+
+	public CreateSmartLockKeyIntegrationTests()
+	{
+		_application = new KerberoWebApplicationFactory<Program>();
+		_application.Server.PreserveExecutionContext = true; // fixture for Flurl
```

Comment:
```suggestion
		_application.Server.PreserveExecutionContext = true; // fixture for Flurl
```
pleas explain what this line do, so we can understand

Reply:
https://github.com/tmenier/Flurl/issues/493#issuecomment-573095892